packages/odyssey-storybook/src/contributing/LabsComponents.stories.mdx

```diff
@@ -0,0 +1,40 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Contributing/Labs Components" />
+
+# Contibuting Odyssey Labs Components
+
+If you are wanting to add components to Odyssey Labs, read this doc first before creating a PR.
+
+## Requirements
+
+Getting components added to Odyssey requires a few steps to be completed first:
+
+1. Get approval from Odyssey design to see if that component even belongs in Odyssey.
+2. The component should be generic enough to allow reuse by all teams at Okta.
```

Comment:
Nit: we may want a separate section for "What belongs in labs" rather than including those criteria as a step for adding an approved component to Labs.